### PR TITLE
Implement a funcitoning tile map for Caleb's zone in the playground and a tool for making template tile maps.

### PR DESCRIPTION
**Pull request by calebkrauter**
- Updated Caleb's zone with a tile map and interpreter to read it in.
- Added a directory for dev tools that we may use for speeding up development like the simple tile map maker I included.

### CODE REVIEW

File: dev_tools_temp/MakeMap.java
Comment (NathanHinthorne) [inline dev_tools_temp/MakeMap.java:1]: Great idea! Nice work creating a whole class for this Caleb.

File: index.html [hunk 1]
```diff
@@ -36,6 +36,8 @@
     <script type="text/javascript" src="./js/classes/vector.js"></script>
     <script type="text/javascript" src="./js/classes/zone.js"></script>
 
+    <script type="text/javascript" src="./js/scripts/tile_maps/caleb_zone_playground.js"></script>
```
Comment (tclaar) [inline index.html:39]: Is this supposed to go under line 23?

File: js/classes/zone.js [hunk 2]
```diff
@@ -130,23 +130,23 @@ class Zone {
...
-                caleb:          new Zone(zeros, defaultMaxBlock, loadPlaygroundCaleb, "Club Caleb"),
-                devin:          new Zone(zeros, defaultMaxBlock, loadPlaygroundDevin, "Devinopolis"),
-                everybody:      new Zone(zeros, defaultMaxBlock, loadPlaygroundEverybody, "Bro City"),
-                nathan:         new Zone(zeros, defaultMaxBlock, loadPlaygroundNathan, "Natesburg"),
-                trae:           new Zone(zeros, defaultMaxBlock, loadPlaygroundTrae, "Trae Town")
+                caleb: new Zone(zeros, defaultMaxBlock, loadPlaygroundCaleb, "Club Caleb"),
+                devin: new Zone(zeros, defaultMaxBlock, loadPlaygroundDevin, "Devinopolis"),
+                everybody: new Zone(zeros, defaultMaxBlock, loadPlaygroundEverybody, "Bro City"),
+                nathan: new Zone(zeros, defaultMaxBlock, loadPlaygroundNathan, "Natesburg"),
+                trae: new Zone(zeros, defaultMaxBlock, loadPlaygroundTrae, "Trae Town")
```
Comment (tclaar) [inline js/classes/zone.js:137]: I think the whitespace was ok to leave in here.

**Review by NathanHinthorne — APPROVED** [review]
Nice changes Caleb. This could really speed up our development process.

File: dev_tools_temp/MakeMap.java
Comment (calebkrauter) [reply dev_tools_temp/MakeMap.java:1]: Thanks!

**Review by tclaar — APPROVED** [review]

**Pull request merged**